benchto-service/src/main/java/com/teradata/benchto/service/BenchmarkService.java

```diff
@@ -126,7 +127,7 @@ public void startExecution(String uniqueName, String benchmarkSequenceId, String
     @Retryable(value = {TransientDataAccessException.class, DataIntegrityViolationException.class}, maxAttempts = 1)
```

Comment:
It is so since ever (commit c266672689e27fdb80baf6345b1e3bcf5d6c7524) and the cmt msg does not contain a clue. This is orthogonal to the change under review, but if you wish, I can remove the `maxAttempts` attributes (default `3` will be used).